src/test/java/org/broadinstitute/hellbender/tools/walkers/qc/PileupIntegrationTest.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.tools.walkers.qc;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class PileupIntegrationTest extends CommandLineProgramTest {
```

Comment:
this test is good but not extensive enough. method `createVerboseOutput` is not tested at all.
Test also the case without a reference